clients/cli/src/main.rs

```diff
@@ -82,9 +96,97 @@ fn main() -> Result<(), Box<dyn Error>> {
         }
         Command::Logout => {
             println!("Logging out and clearing node configuration file...");
-            let config_path = get_config_path().expect("Failed to get config path");
             Config::clear_node_config(&config_path).map_err(Into::into)
         }
+        Command::RegisterUser {
+            env,
+            wallet_address,
+        } => {
+            let environment = env.unwrap_or_default();
+            println!(
+                "Registering user with wallet address: {} in environment: {:?}",
+                wallet_address, environment
+            );
+            // Check if the wallet address is valid
+            if !keys::is_valid_eth_address(&wallet_address) {
+                let err_msg = format!(
+                    "Invalid Ethereum wallet address: {}. It should be a 42-character hex string starting with '0x'.",
```

Comment:
This is a purely syntactic check. Are we able to/would we want to check existence in some way?

Reply:
Good point. 
At least for now, I think this is enough validation to provide fail-fast behavior, backstopped by the orchestrator's checks when registering an new user.